gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -537,6 +537,7 @@ def main(args):
                 raw_z_outlier_threshold_upper_syn=args.raw_z_outlier_threshold_upper_syn,
                 include_os=int(version[0])
                 < 4,  # OS (other splice) is not implemented for build 38.
+                use_mane_select_instead_of_canonical=int(version[0]) >= 4,
```

Comment:
one thing I just noticed (sorry missed this earlier) is that the code checks for version happens twice:
```
int(version[0]) < 4
```
and
```
int(version[0])) >= 4
```
gets used more than once. maybe this should get set as variable earlier on, above the call to `compute_constraint_metrics`? something like
```
version_ge_4 = int(version[0]) >= 4
```
and then
```
include_os=not version_ge_4,
use_mane_select_instead_of_canonical=version_ge_4,
```